torch/csrc/inductor/aoti_torch/c/shim_mps.h

```diff
@@ -0,0 +1,26 @@
+#ifndef AOTI_TORCH_SHIM_MPS
+#define AOTI_TORCH_SHIM_MPS
+
+#include <ATen/native/mps/MetalShaderLibrary.h>
+#include <torch/csrc/inductor/aoti_torch/c/shim.h>
+
+typedef struct {
+  std::shared_ptr<at::native::mps::MetalKernelFunction> kernelFunction;
```

Comment:
I don't think we should directly refer to aten struct here?

Reply:
Should I be using a void* instead?

Reply:
It should be similar to AtenTensorOpaque https://github.com/pytorch/pytorch/blob/500a710422f8b587f154dcfe9c4fe860ef50fef1/torch/csrc/inductor/aoti_torch/c/shim.h#L86, and reinterpret AOTIMetalKernelFunctionHandle when used in .cpp files.

Reply:
Updated!